frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -63,8 +63,9 @@ export const useResolvedExtensions: UseResolvedExtensions = require('@console/dy
  * A component that creates a Navigation bar for a page.
  * Routing is handled as part of the component.
  * `console.tab/horizontalNav` can be used to add additional content to any horizontal nav.
- * @param {object} [resource] - the resource associated with this Navigation, an object of K8sResourceCommon type
+ * @param {object} [resource] - (optional) the resource associated with this Navigation, an object of K8sResourceCommon type
  * @param {NavPage[]} pages - an array of page objects
+ * @param {*} [customData] - (optional) custom data to be passed to the horizontal nav
```

Comment:
We need more documentation about how/when this would be used, ideally with examples.